format-specs/geoparquet.md

```diff
@@ -37,9 +37,9 @@ All file-level metadata should be included under the "geo" key in the parquet me
 
 |     Field Name     |  Type  |                             Description                              |
 | ------------------ | ------ | -------------------------------------------------------------------- |
-| version     		 | string | **REQUIRED** The version of the GeoParquet metadata standard used when writing.   |
-| primary_column     | string | **REQUIRED** The name of the "primary" geometry column.                |
-| columns            | object<key, [Column Metadata](#column-metadata)> | **REQUIRED** Metadata about geometry columns, with each key is the name of a geometry column in the table. |
+| version     		 | string | **REQUIRED** The version of the GeoParquet metadata standard used when writing. |
+| primary_column     | string | **REQUIRED** The name of the "primary" geometry column with a minimum length of 1. |
```

Comment:
I don't know if it's necessarily need to mention it here as well (I find it somewhat obvious that a column name is not an empty string), but either way is fine.

Reply:
```suggestion
| primary_column     | string | **REQUIRED** The name of the "primary" geometry column |
```